1.9.0

```diff
@@ -258,6 +258,40 @@
           }
         }
       ]
+    },
+    {
+      "sobjectName":"Campaign",
+      "instructionTypeName":"mdTaskListView",
+      "taskList":[
+        {
+          "jiraTicket": "AC-59235",
+          "description": "Delete AllActiveCampaigns List View",
+          "actionType": "deleteListView",
```

Comment:
This is resolved in the mdTaskFactory. Should be just 'ListView' and the factory class will prepend 'mdTask' then try to see if there is a class named 'mdTask' + actionType.

Reply:
removed delete